src/components/__tests__/Bullet.ts

```diff
@@ -126,6 +126,22 @@ describe('render', () => {
     const bullets = document.querySelectorAll('.bullet')
     expect(bullets.length).toBe(4)
   })
+
+  // when hidden thoughts are enabled, we render a parent bullet for thoughts with meta attribute children
+  it('renders a parent bullet when hidden thoughts are enabled', async () => {
+    await dispatch([
+      importText({
+        text: `
+        - A
+          - =test
+      `,
+      }),
+      toggleHiddenThoughts(),
+    ])
+
+    const bullets = document.querySelectorAll('.triangle')
```

Comment:
Can you add a `data-bullet-leaf` attribute to test against? I would like to start decoupling testing from CSS.

Reply:
Good call. I've introduced a `data-bullet` attribute that is either `data-bullet="parent"` or `data-bullet="leaf"`, to cover both cases in one attribute.